paper/paper.md

```diff
@@ -0,0 +1,116 @@
+*starfish* is built on top of popular Python tools like xarray (@xarray) and scikit-image (@scikit-image).
+
+To help new users get started and support the broader single cell biology community in learning how to work with these data, *starfish* maintains example datasets and reference implementations ported from published assays, including
+MERFISH (@MERFISH),
+In Situ Sequencing (@ISS),
+osmFISH (@osmFISH),
+BaristaSeq (@BaristaSeq),
+smFISH,
+DARTFISH,
+STARmap (@starmap),
+and seqFISH (@seqFISH).
+
+*starfish* was developed alongside the [SpaceTx project](https://spacetx-starfish.readthedocs.io/en/stable/about/index.html), a CZI-funded effort to compare spatial transcriptomics methods in the context of determining cell types in the brain.
+*starfish* is currently in use by multiple research groups, including the [Allen Institute for Brain Science](https://alleninstitute.org), the [Chan Zuckerberg Biohub](https://www.czbiohub.org/), and the [Zhang Lab at UC San Diego](http://jinzhanglab.ucsd.edu/).
+These groups support multiple large-scale projects profiling *in situ* gene expression, including the SpaceTx consortium, the [Human Cell Atlas](https://www.humancellatlas.org/), the [BRAIN Initiative Cell Census Network](https://biccn.org/), and the [HuBMAP Consortium](https://hubmapconsortium.org/).
+
+
+# Acknowledgements
+
+We would like to acknowledge the following contributions, which have been invaluable to the development of starfish.
+
+For direct contributions of code and documentation, we thank
+Olga Botvinnik,
+Gökçen Eraslan,
+Kira Evans,
+Marcus Kinsella,
+Nicholas Mei,
+Josh Moore,
+Nicholas Sofroniew,
+and Ola Tarkowska.
+
+We would like to thank the members of the SpaceTx consortium for working closely with us to understand their image processing steps, port pipelines into starfish, and contribute example datasets.
+We especially would like to thank
+Jeff Moffitt and Xiaowei Zhuang for their help with the MERFISH pipeline and example data,
+Marco Mignardi and Mats Nilsson for their help with the In Situ Sequencing pipeline and example data,
+Simone Codeluppi and Sten Linnarsson for their help with the osmFISH pipeline and example data,
+Xioayin Chen and Anthony Zador for their help with the BaristaSeq pipeline and example data,
+Richard Que and Kun Zhang for their help with the DARTFISH pipeline and example data,
+Xiao Wang, William Allen, and Karl Deisseroth for their help with the STARmap pipeline and example data,
+Dan Goodwin and Ed Boyden for their help with the ExFISH pipeline and example data,
+Nico Pierson, Sheel Shah, and Long Cai for their help with the seqFISH pipeline and example data,
+and Denis Shapiro for their help with the Imaging Mass Cytometry example data.
+
+Finally, we would like to thank the SciTech team and Single Cell Program at the Chan Zuckerberg Initiative for their support of this work.
```

Comment:
`SciTech` is jargon, maybe `Science and Technology teams`?

Reply:
good call.